README.md

```diff
@@ -1,43 +1,61 @@
 # ViewComponent::Form
 
-**ViewComponent::Form** provides a `FormBuilder` with the same interface as [`ActionView::Helpers::FormBuilder`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html), but using [ViewComponent](https://github.com/github/view_component)s for rendering the fields. It's a starting point for writing your own custom ViewComponents.
-
-:warning: **This is an early release: the API is subject to change until we reach `v1.0.0`.**
+**`ViewComponent::Form`** is a customizable form builder using the same interface as [`ActionView::Helpers::FormBuilder`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html) but with extensible [ViewComponent](https://github.com/github/view_component) components.
 
 Development of this gem is sponsored by:
 
 <a href="https://etamin.studio/?ref=view_component-form"><img src="https://etamin.studio/images/logo.svg" alt="Sponsored by Etamin Studio" width="184" height="22"></a>      <a href="https://pantographe.studio/?ref=view_component-form"><img src="https://static.s3.office.pantographe.cloud/logofull.svg" alt="Sponsored by Pantographe" width="156" height="25"></a>
 
 ## Compatibility
 
+> [!WARNING]  
+> **This is an early release, and the API is subject to change until `v1.0.0`.**
+
 This gem is tested on:
+
 - Rails 6.0+ (with or without ActionText)
 - Ruby 2.7+
 
 ## Installation
 
-Add this line to your application's Gemfile:
-
-```ruby
-gem 'view_component-form'
+```shell
+bundle add view_component-form
 ```
 
-And then execute:
+### Configuration
```

Comment:
I think this bit comes too early (or need more info on how to set up this config).

Maybe we can add this as a usage example:

```rb
# config/initializers/vcf.rb

ViewComponent::Form.configure do |config|
  config.parent_component = 'ApplicationFormComponent'
end
```

Reply:
Resolved via 8e545ad